frontend/src/views/PreparationView.tsx

```diff
@@ -0,0 +1,80 @@
+
+import * as React from 'react';
+import { Chip, NavigationStructure, NavigationDrawer } from '@equinor/fusion-components';
+import { Barrier, Answer, Evaluation, Question, } from '../api/models';
+import QuestionAndAnswerForm from '../components/QuestionAndAnswer/QuestionAndAnswerForm';
+import { Box } from '@material-ui/core';
+
+interface BarrierQuestionsProps
+{
+    title: string
+    questions: Question[]
+}
+
+const BarrierQuestions = ({title, questions}: BarrierQuestionsProps) => {
+    return (
+        <>
+            <h2>{title}</h2>
```

Comment:
Should we rather use eds `Typography`here?

```suggestion
            <Typography variant="h2">{title}</Typography>
```